src/Number.php

```diff
@@ -115,4 +115,36 @@ public static function fromFloat($floatingPoint)
 
         return new self(sprintf('%.8g', $floatingPoint));
     }
+
+    /**
+     * @return bool
+     */
+    public function isNegative()
+    {
+        return $this->number[0] === '-';
+    }
+
+    /**
+     * @return string
+     */
+    public function getIntegerPart()
```

Comment:
Regarding the terminology: I am basing in on [this thread on StackExchange](http://math.stackexchange.com/questions/3448/rules-for-rounding-positive-and-negative-numbers).